src/utils/hooks/useLocalStorage.ts

```diff
@@ -51,5 +51,17 @@ export default function useLocalStorage(key: string, initialValue: object = {})
     window.localStorage.setItem(localStorageKey, JSON.stringify(jsonValue));
   };
 
-  return [storedValue, setValue];
+  const clearValue = () => {
+    if (objectKey) {
+      const newObject = {
+        [objectKey]: initialValue,
+      };
+      window.localStorage.setItem(localStorageKey, JSON.stringify(newObject));
```

Comment:
tok litt tid før jeg skjønte hvorfor man skulle lagre et nytt objekt i localstorage i denne funksjonen, men skjønner at newObject egentlig er det initielle objektet som er tomt. burde kanskje kalt det noe annet enn newObject

Reply:
Ser at denne faktisk vil slette data for andre forms også. Endrer til å bare kalle `setValue(initialValue)`. Da vil key aldri slettes fra localStorage, men ser heller ingen særlig ulempe med det per nå. Tror uansett vi må gå flere runder med persistering til localStorage

Reply:
lurt 👍  må nok flikkes en del på den der ja